### PR TITLE
Cypress tests: preset the admin API key to a static value

### PR DESCRIPTION
**Pull request by arikfr**
(this to avoid diffs in screenshots)

@gabrieldutra FYI.

### CODE REVIEW

File: cypress/cypress.js [hunk 1]
```diff
@@ -1,72 +1,87 @@
...
+    console.log("POST " + request.route + " - " + result);
     if (seedValues.length) {
       seedDatabase(seedValues);
     }
   });
+
+  // Make sure the admin user has the same API key on every execution
+  execSync(
+    "docker-compose -p cypress run postgres psql -h postgres -c \"update users set api_key = 'secret' where email ='admin@redash.io';\""
+  );
```
Comment (arikfr) [inline cypress/cypress.js]: This is the actual test. The rest is Prettier doing its thing.

Comment (gabrieldutra) [reply cypress/cypress.js]: Nice! That was one of the solutions I thought about (the other one would require me to use Cypress Stubs).

My idea is to create a few Cypress tests in #3354, probably one for regenerating the API Key. If needed in there, I'll revisit this.

BTW, I thought the pattern was to use singlequoted :sweat_smile:

Comment (arikfr) [reply cypress/cypress.js]: > BTW, I thought the pattern was to use singlequoted 😅

🤔 you're correct. Something went wrong there. I'll fix.

File: Dockerfile [hunk 2]
```diff
@@ -14,7 +14,7 @@ ARG skip_ds_deps
 
 # We first copy only the requirements file, to avoid rebuilding on every file
 # change.
-COPY requirements.txt requirements_dev.txt ./
+COPY requirements.txt requirements_dev.txt requirements_all_ds.txt ./
```
Comment (gabrieldutra) [inline Dockerfile]: :eyes:

Comment (arikfr) [reply Dockerfile]: https://github.com/getredash/redash/pull/3358/commits/ec9117c6e53b727e841da2d041badbac8b852065

**Pull request merged**